lib/stewarding.ex

```diff
@@ -19,14 +19,14 @@ defmodule Stewarding do
   end
 
   def add_person(key) do
-    Stewarding.Repo.insert %Stewarding.Person{key: key}
+    Stewarding.Person.add_person(key)
```

Comment:
`Person.add_person` is quite redundant. Can probably rename this to `add` or `add_entry` if that method even fits in the `Stewarding.Person` module in the first place.